src/bin/pg_autoctl/monitor.c

```diff
@@ -1654,7 +1654,7 @@ monitor_print_state(Monitor *monitor, char *formation, int group)
 		{
 			sql =
 				"SELECT * FROM pgautofailover.current_state($1,$2) "
-				"ORDER BY node_id";
+				"ORDER BY nodename";
```

Comment:
Same. Can we avoid doing that?

Reply:
I guess we could make node3 wait on node2 being up. The reason I did this was that it's quite confusing to have node2 be the third node in the list.